rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -26,63 +26,72 @@ Create and document the environment for the automation of processes.
 
-A project can only be assigned to a single category. First create a standard category, so you can start with process automation quickly. You can change the category of a process later. After you have automated the first projects, it makes sense to create an archive category and move completed projects there.
+A project can only belong to a single category. First, create a standard category, so you can start with process automation quickly. You can change the category of a process later. After you have automated the first projects, create an archive category and move completed projects there.
 
 ==== Create a Category
 
-Create categories to group processes. Assign a process to a category during evaluation or project creation.
+Create categories to group processes.
+
+Assign a process to a category during evaluation or project creation.
 
-. In the *Process Management* view of the *Process Automation* module, click *Create*.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click *Create*.
 . Enter a name in the *Create a category* window.
 . Click *OK*.
 
 ==== Edit a Category
 
 Edit a category to render its name more precisely.
 
-. In the *Process Management* view of the *Process Automation* module, click the *Edit* icon image:edit-icon.png[pen-to-square symbol,1.5%,1.5%] on the panel of the category you want to edit.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click the *Edit* icon image:edit-icon.png[pen-to-square symbol,1.5%,1.5%] on the panel of the category you want to edit.
 . Change the name in the *Edit the category* window.
 . Click *OK*.
 
 ==== Remove a Category
 
 Remove categories you no longer need.
 
-Only categories that do not contain processes can be deleted.
+You can only remove categories that do not contain processes.
 
-. In the *Process Management* view of the *Process Automation* module, click the the *Delete* icon image:delete-icon.png[trash symbol,1.5%,1.5%] on the panel of the category you want to delete.
+. Open the *Process Management* view of the *Process Automation* module.
+. Click the the *Remove* icon image:delete-icon.png[trash symbol,1.5%,1.5%] on the panel of the category you want to delete.
 
 === Manage Applications
 
-In the view *Process Applications*, list all applications required for the execution of processes. In the view *Process Management*, assign required applications to processes at process creation or phase transition. In the module *Bot Management*, assign applications to a bot that can execute them. Bots can execute all applications installed on their computer.
+In the view *Process Applications*, you list all applications required for the execution of processes.
+
+In the view *Process Management*, the project manager xref:myrpa-start.adoc#required-process-applications[assigns required applications to a process] at process creation or phase transition.
```

Comment:
Suggestion to put the entity type after the entity name here and elsewhere: 
view *Process Management* > *Process Management* view. This reads better in English, but what you have is acceptable if you don't want to change it.